fusestore/store/src/executor/action_handler_test.rs

```diff
@@ -65,6 +77,64 @@ async fn test_action_handler_do_pull_file() -> anyhow::Result<()> {
     Ok(())
 }
 
+struct D {
+    plan: CreateDatabasePlan,
+    want: common_exception::Result<CreateDatabaseActionResult>,
+}
+
+/// helper to build a D
+fn case_db(db_name: &str, if_not_exists: bool, want: common_exception::Result<u64>) -> D {
```

Comment:
As a nested type I gave it a short name because it is unlikely to have a name conflict with others.
As a mod-level struct, a single letter `D` is too short as a name.
I'd suggest keeping it a nested type or giving it an explicit and meaningful name.